### PR TITLE
minor copy edits on sample applications

### PR DESCRIPTION
**Pull request by genwhittTTD**
minor copy edits on sample applications

### CODE REVIEW

File: examples/google-secure-signals-integration/with_sdk_v3/views/intro.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <h1>Server-Side Integration Example, UID2 JavaScript SDK</h1>
 <p class="intro">
   This example demonstrates how a content publisher can use the UID2 services and the
-  <a href="https://unifiedid.com/docs/sdks/client-side-identity"
-    >Client-Side Identity JavaScript SDK</a
+  <a href="https://unifiedid.com/docs/sdks/sdk-ref-javascript"
+    >UID2 SDK for JavaScript</a
```
Comment (sunnywu) [inline examples/google-secure-signals-integration/with_sdk_v3/views/intro.html:5]: Suggest we do the same thing and explain:

> This includes:
> 
> 1.     Generate tokens on the server side using v2/token/generate endpoint
> 2.    Pass the token value back to the UID2 SDK For Javascript embedded in the client web page and make it available to be retrieved by the Google Publisher Tag (GPT) javascripts.

**Review by sunnywu — APPROVED** [review]

File: examples/google-secure-signals-integration/with_sdk_v3/views/intro.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <h1>Server-Side Integration Example, UID2 JavaScript SDK</h1>
```
Comment (sunnywu) [inline examples/google-secure-signals-integration/with_sdk_v3/views/intro.html:1]: this should be Client-Server based on our slack chat with @ssundahlTTD

**Review by ssundahlTTD — APPROVED** [review]

File: examples/google-secure-signals-integration/server_only/views/header.html [hunk 3]
```diff
@@ -26,11 +26,11 @@
...
+    <h1>Server-Side UID2 Integration Example</h1>
     <p class="intro">
       This example demonstrates how a content publisher can use the UID2 services to implement the
-      <a href="https://unifiedid.com/docs/guides/google-ss-integration#server-only-integration"
-        >Server-Only Integration</a
+      <a href="https://unifiedid.com/docs/guides/integration-google-ss#server-side-integration"
+        >Server-Side Integration</a
       >. [<a
         href="https://github.com/IABTechLab/uid2-web-integrations/tree/main/examples/google-secure-signals-integration/server_only"
         >Source Code</a
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply examples/google-secure-signals-integration/server_only/views/header.html:36]: To be addressed in a later PR, per discussion.

File: examples/google-secure-signals-integration/with_sdk_v3/views/intro.html (hunk 1 above)
Comment (genwhittTTD) [reply examples/google-secure-signals-integration/with_sdk_v3/views/intro.html:5]: To be addressed in a later PR, per discussion.

**Pull request merged**